packages/runtime/client-api/src/api/document.ts

```diff
@@ -90,6 +83,7 @@ export class Document extends EventEmitter {
         public readonly context: IFluidDataStoreContext,
         private readonly root: ISharedMap,
         private readonly closeFn: () => void,
+        public readonly existing: boolean,
```

Comment:
Can we remove it? :)

Reply:
There are some references to it unfortunately. I'll check if I can refactor them.